src/full/Agda/TypeChecking/Monad/Base.hs

```diff
@@ -3334,6 +3337,13 @@ isDontExpandLast ExpandLast           = False
 isDontExpandLast DontExpandLast       = True
 isDontExpandLast ReallyDontExpandLast = True
 
+isExpandLast :: ExpandHidden -> Bool
+isExpandLast = not . isDontExpandLast
```

Comment:
I would directly define this and then `isDontExpandLast = not . isExpandLast`.

Reply:
done